packages/design-system/.gitignore

```diff
@@ -0,0 +1,26 @@
+# Logs
```

Comment:
similarly - it's not worth to keep .gitignore in each package, u can configure it "globally" in the root - unless there are some package-level stuff you want to ignore, but in here most of the stuff is generic (like logs, dist, node_modules etc)

Reply:
moved